jgeorge37/acmwosu-db
# Jcg/ada 11 signin logic

To test:

1. Create a local PostgreSQL instance (see README). Make sure you add the pgcrypto extension.
2. Change the database URL in the inner .env file to be your local database URL
3. Using probably the psql shell, insert a new row into the account table. This should have the email and the password for a test account. Encrypt the password like so, using the md5 salt thing: `INSERT INTO account (email, password) VALUES ('testEmail@test.com', crypt('testpw', gen_salt('md5')));` 
4. Check that the above worked by using `SELECT * FROM account;`, there should be the email you chose and a jumbled password.
5. Comment out the line that says comment out if using local DB in pg-query.js
6. npm run dev and go to the sign in page. Try to sign in with a bad email/pw combo; there should be a red error message above the submit button
7. sign in with your good user/pw combo, the error message should go away.
